src/tests/ftest/io/romio.py

```diff
@@ -100,7 +100,7 @@ def test_romio(self):
         Test ID: DAOS-1994
         Run Romio test provided in mpich package
         Testing various I/O functions provided in romio test suite
-        :avocado: tags=mpio,romio
+        :avocado: tags=mpio,romio,vm,regression
```

Comment:
Not appropriate to add this to regression at this time.  Romio test requires the romio test already be present on the CI machines which it is not.  Brian has a ticket to do this work but kind of middle of the pack priority.

Reply:
I added this, as my code was based on the romio test and didn't work, I wanted to check that the romio test itself actually worked, and as you say it failed because the binary isn't found.  This isn't intended to be merged with this change in place.